webpack/assets/javascripts/react_app/components/notifications/NotificationsDrawerHelpers.js

```diff
@@ -0,0 +1,14 @@
+import { documentLocale } from '../../common/I18n';
+
+export const formatDateTime = datetime => {
+  const locale = documentLocale().replace(/-/g, '_');
```

Comment:
```suggestion
  const locale = documentLocale()
  const options = {
    month: 'numeric',
    day: 'numeric',
    year: 'numeric',
    hour: 'numeric',
    minute: 'numeric',
    second: 'numeric',
    hour12: true,
  }
  try {
  return new Date(datetime).toLocaleDateString(locale, options)
  } catch {
   return new Date(datetime).toLocaleDateString('en', options)
  }
```
The replace is wrong sorry, the code I took it from has a try catch so it didnt crash there.
So to avoid crashing I think this should be changed like this, this way it wont crash when users select 'en-GB'